js/translations.js

```diff
@@ -1,25 +1,46 @@
 function updateContent() {
   update();
-  $("body").localize();
+  $('body').localize();
 }
-
+const defaultLanguage = 'en';
 i18next
-  .use(i18nextXHRBackend)
-  .use(i18nextBrowserLanguageDetector)
-  .init({
-    fallbackLng: 'en',
-    debug: true,
-    backend: {
-      loadPath: 'locales/{{lng}}.json',
-    },
-  }, function(err, t) {
-    jqueryI18next.init(i18next, $);
-    i18next.on('languageChanged', () => {
-      updateContent();
-    });
-
-    // init set content
-    $(document).ready(initialize);
-    $(document).on("input", updateContent);
-    $('input[type = radio]').on("change", updateContent);
+.use(i18nextXHRBackend)
+.use(i18nextBrowserLanguageDetector)
+.init({
+  fallbackLng: defaultLanguage,
+  debug: true,
+  backend: {
+    loadPath: 'locales/{{lng}}.json',
+  },
+}, (err, t) => {
+  const languages = [
+    ['de', 'Deutsch'],
+    ['en', 'English'],
+    ['zh-CN', '简体中文'],
+    ['zh-TW', '繁體中文']
+  ],
```

Comment:
Should this be `;`?

Reply:
The `languageSelector` is also defined, so it is correct. 
```suggestion
  const languages = [
    ['de', 'Deutsch'],
    ['en', 'English'],
    ['zh-CN', '简体中文'],
    ['zh-TW', '繁體中文']
  ],
  languageSelector = $('#language');
```